src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,59 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final byte MAX_ITEMS_NUMBER = 10;
+    private final K[] arrayKeys;
+    private final V[] arrayValues;
+    private byte index = 0;
+    private boolean nullMarker = false;
```

Comment:
```suggestion
    private int size;
```
You don't need this variable, let me explain how to simplify your code

Reply:
Thank you, it means a lot to me